iamclient/iamclient.go

```diff
@@ -144,6 +149,11 @@ func New(
 	return client, nil
 }
 
+// GetNodeID returns node ID.
+func (client *Client) GetNodeID() string {
+	return client.nodeID
```

Comment:
should we return error if node id was not taken from IAM?

Reply:
iamclient will be not created, New returns error in this case.